data/aliases.js

```diff
@@ -826,29 +826,801 @@ let BattleAliases = {
+	"utsubotto": "Victreebel",
+	"menokurage": "Tentacool",
+	"dokukurage": "Tentacruel",
+	"ishitsubute": "Geodude",
+	"goroun": "Graveler",
+	"gorounya": "Golem",
+	"ponita": "Ponyta",
+	"gyaroppu": "Rapidash",
+	"yadon": "Slowpoke",
+	"yadoran": "Slowbro",
+	"koiru": "Magnemite",
+	"reakoiru": "Magneton",
+	"kamonegi": "Farfetch'd",
+	"doudou": "Doduo",
+	"doudorio": "Dodrio",
+	"pauwau": "Seel",
+	"jugon": "Dewgong",
+	"betobetaa": "Grimer",
+	"betobeton": "Muk",
+	"sherudaa": "Shellder",
+	"parushen": "Cloyster",
+	"gousu": "Gastly",
+	"gousuto": "Haunter",
+	"gengaa": "Gengar",
+	"iwaaku": "Onix",
+	"suripu": "Drowzee",
+	"suripaa": "Hypno",
+	"kurabu": "Krabby",
+	"kinguraa": "Kingler",
+	"biriridama": "Voltorb",
+	"marumain": "Electrode",
+	"tamatama": "Exeggcute",
+	"nasshi": "Exeggutor",
+	"karakara": "Cubone",
+	"garagara": "Marowak",
+	"sawamuraa": "Hitmonlee",
+	"ebiwaraa": "Hitmonchan",
+	"beroringa": "Lickitung",
+	"dogaasu": "Koffing",
+	"matadogasu": "Weezing",
+	"saihoun": "Rhyhorn",
+	"saidon": "Rhydon",
+	"rakki": "Chansey",
+	"monjara": "Tangela",
+	"garuura": "Kangaskhan",
+	"tattsuu": "Horsea",
+	"shidora": "Seadra",
+	"tosakinto": "Goldeen",
+	"azumaou": "Seaking",
+	"hitodeman": "Staryu",
+	"sutaami": "Starmie",
+	"bariyaado": "Mr. Mime",
+	"sutoraiku": "Scyther",
+	"ruujura": "Jynx",
+	"erebuu": "Electabuzz",
+	"buubaa": "Magmar",
+	"kairosu": "Pinsir",
+	"kentarosu": "Tauros",
+	"koikingu": "Magikarp",
+	"gyaradosu": "Gyarados",
+	"rapurasu": "Lapras",
+	"metamon": "Ditto",
+	"bui": "Eevee",
+	"shawaazu": "Vaporeon",
+	"sandaasu": "Jolteon",
+	"buusutaa": "Flareon",
+	"porigon": "Porygon",
+	"omunaito": "Omanyte",
+	"omusutaa": "Omastar",
+	"kabutopusu": "Kabutops",
+	"putera": "Aerodactyl",
+	"kabigon": "Snorlax",
+	"furizaa": "Articuno",
+	"sandaa": "Zapdos",
+	"faiyaa": "Moltres",
+	"miniryuu": "Dratini",
+	"hakuryuu": "Dragonair",
+	"kairyuu": "Dragonite",
+	"myuutsuu": "Mewtwo",
+	"myuu": "Mew",
+	"beirifu": "Bayleef",
+	"meganiumu": "Meganium",
+	"hinoarashi": "Cyndaquil",
+	"magumarashi": "Quilava",
+	"bakufuun": "Typhlosion",
+	"waninoko": "Totodile",
+	"arigeitsu": "Croconaw",
+	"dairu": "Feraligatr",
+	"otachi": "Sentret",
+	"tachi": "Furret",
+	"houhou": "Hoothoot",
+	"yorunozuku": "Noctowl",
+	"rediba": "Ledyba",
+	"redian": "Ledian",
+	"itomaru": "Spinarak",
+	"ariadosu": "Ariados",
+	"kurobatto": "Crobat",
+	"chonchi": "Chinchou",
+	"rantaan": "Lanturn",
+	"pichuu": "Pichu",
+	"pi": "Cleffa",
+	"pupurin": "Igglybuff",
+	"togechikku": "Togetic",
+	"neiti": "Natu",
+	"neitio": "Xatu",
+	"meripu": "Mareep",
+	"mokoko": "Flaaffy",
+	"denryuu": "Ampharos",
+	"kireihana": "Bellossom",
+	"mariru": "Marill",
+	"mariruri": "Azumarill",
+	"usokki": "Sudowoodo",
+	"nyorotono": "Politoed",
+	"hanekko": "Hoppip",
+	"popokko": "Skiploom",
+	"watakko": "Jumpluff",
+	"eipamu": "Aipom",
+	"himanattsu": "Sunkern",
+	"kimawari": "Sunflora",
+	"yanyanma": "Yanma",
+	"upaa": "Wooper",
+	"nuou": "Quagsire",
+	"fi": "Espeon",
```

Comment:
Espeon is `eefi` in Japanese (official romaji `eifie`)

Reply:
Oh, this one is a programmer error - Ruby's downcase doesn't seem to be unicode friendly, it the capital with an accent got stripped.